apiserver/admin.go

```diff
@@ -65,6 +65,12 @@ func (a *admin) login(req params.LoginRequest, loginVersion int) (params.LoginRe
 
 	if a.root.state.IsCAAS() {
 		// XXX don't bother with actual login checks here. Just fake it for the prototype for now.
+		u, err := a.srv.state.User(names.NewUserTag("admin"))
```

Comment:
Sorry, you're right. It would have worked (but with the wrong entity loaded). Using the wrong entity is likely to bite us later when we forget :)